### PR TITLE
Added stylecop ruleset, directory.build.props files, fixed formatting and code-style issues

### PR DESCRIPTION
**Pull request by Hellevar**

### CODE REVIEW

File: src/TestEnvironment.Docker.Containers.Elasticsearch/TestEnvironment.Docker.Containers.Elasticsearch.csproj [hunk 1]
```diff
@@ -2,19 +2,14 @@
...
     <AssemblyName>TestEnvironment.Docker.Containers.Elasticsearch</AssemblyName>
     <RootNamespace>TestEnvironment.Docker.Containers.Elasticsearch</RootNamespace>
     <Version>1.0.5</Version>
     <PackageId>TestEnvironment.Docker.Containers.Elasticsearch</PackageId>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <Authors>Aliaksei Harshkalep</Authors>
     <Description>Add Elasticsearch container specific functionality.</Description>
     <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
     <PackageTags>docker tests elasticsearch container</PackageTags>
-    <PackageProjectUrl>https://github.com/Deffiss/testenvironment-docker</PackageProjectUrl>
```
Comment (KirylKiryanchykau) [inline src/TestEnvironment.Docker.Containers.Elasticsearch/TestEnvironment.Docker.Containers.Elasticsearch.csproj:15]: Should the staff like that be removed?

Comment (Hellevar) [reply src/TestEnvironment.Docker.Containers.Elasticsearch/TestEnvironment.Docker.Containers.Elasticsearch.csproj:15]: This value was duplicated across all project files, so i extracted it to Directory.Build.props.

File: src/TestEnvironment.Docker.Containers.Elasticsearch/ElasticsearchContainerCleaner.cs [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
 
 namespace TestEnvironment.Docker.Containers.Elasticsearch
 {
     public class ElasticsearchContainerCleaner : IContainerCleaner<ElasticsearchContainer>
     {
         public async Task Cleanup(ElasticsearchContainer container, CancellationToken token = default)
         {
-            if (container == null) throw new ArgumentNullException(nameof(container));
+            if (container == null)
+            {
```
Comment (Deffiss) [inline src/TestEnvironment.Docker.Containers.Elasticsearch/ElasticsearchContainerCleaner.cs:13]: My personal preference is to leave this in one line.

Comment (Deffiss) [reply src/TestEnvironment.Docker.Containers.Elasticsearch/ElasticsearchContainerCleaner.cs:13]: Ok, agree to leave brackets

**Pull request merged**